backend/api/EventHandlers/InspectionFindingEventHandler.cs

```diff
@@ -103,54 +103,51 @@ private async Task<List<Finding>> GenerateFindingsList(List<InspectionFinding> i
+                    new JObject(new JProperty("name", "Anlegg"), new JProperty("value", finding.PlantName)),
+                    new JObject(new JProperty("name", "Område"), new JProperty("value", finding.AreaName)),
+                    new JObject(new JProperty("name", "Tag Number"), new JProperty("value", finding.TagId)),
+                    new JObject(new JProperty("name", "Beskrivelse"), new JProperty("value", finding.FindingDescription)),
+                    new JObject(new JProperty("name", "Tidspunkt"), new JProperty("value", finding.Timestamp.ToString("yyyy-MM-dd HH:mm:ss", CultureInfo.InvariantCulture)))
+                );
+
+                var findingObj = new JObject(
+                    new JProperty("activityTitle", $"Finding ID: \"{finding.TagId}\""),
+                    new JProperty("facts", factsArray));
+                findingsJsonArray.Add(findingObj);
             }
-            return reportBuilder.ToString();
-        }
 
-        public static string GenerateAdaptiveCard(string messageContent, ILogger<InspectionFindingEventHandler> logger)
-        {
-            logger.LogInformation("Message content for Adaptive card: {message}", messageContent);
-            string adaptiveCardJson = $@"{{
-                        ""type"": ""message"",
-                        ""attachments"": [
-                            {{
-                                ""contentType"": ""application/vnd.microsoft.card.adaptive"",
-                                ""content"": {{
-                                    ""type"": ""AdaptiveCard"",
-                                    ""body"": [
-                                        {{
-                                            ""type"": ""Container"",
-                                            ""height"": ""stretch"",
-                                            ""items"": [
-                                                {{
-                                                    ""type"": ""TextBlock"",
-                                                    ""text"": ""{messageContent}"",
-                                                    ""wrap"": true
-                                                }}
-                                            ]
-                                        }}
-                                    ],
-                                    ""$schema"": ""http://adaptivecards.io/schemas/adaptive-card.json"",
-                                    ""version"": ""1.0"",
-                                    ""msteams"": {{
-                                        ""displayStyle"": ""full""
-                                    }}
-                                }}
-                            }}
-                        ]
-                    }}";
+            var sections = new JArray(
+                new JObject(
+                    new JProperty("activityTitle", $"Inspection report for \"{findingsReports[0].PlantName}\""),
```

Comment:
We do we use the [0] and not [i]?

Reply:
Well it will always be the same plant per report but should be i I agree :)

Reply:
Well this is actually outside the for loop :)

Reply:
Have a for each loop now instead, keeping this one as is or this line can also be removed later.